librosa/util/decorators.py

```diff
@@ -4,9 +4,10 @@
 '''Helpful tools for deprecation'''
 
 import warnings
+from packaging import version
```

Comment:
Can you add `packaging` to the requirements list in setup.py?

Otherwise looks good!